pkg/constants/constants.go

```diff
@@ -0,0 +1,48 @@
+package constants
+
+const (
+
+	// JobSetKind is the Kind name for the JobSet.
+	JobSetKind string = "JobSet"
+
+	// JobCompletionIndexFieldPath is the field path for the Job completion index annotation.
+	JobCompletionIndexFieldPath string = "metadata.annotations['batch.kubernetes.io/job-completion-index']"
+
+	// JobTrainerNode is the Job name for the trainer node.
+	JobTrainerNode string = "trainer-node"
```

Comment:
What is the motivation of `trainer-` prefix? Do you aim to add "node" role to other components like modelInitializer?

Reply:
Not right now, but I think this make clearer that this ReplicatedJob is doing training, not initialization, coordination, or post-processing